include/sys/dkioc_free_util.h

```diff
@@ -48,7 +48,7 @@ typedef struct dkioc_free_list_s {
 } dkioc_free_list_t;
 
 static inline void dfl_free(dkioc_free_list_t *dfl) {
-	kmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
+	vmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
```

Comment:
Then let's add one.  We're already forced to diverge slightly here, let's diverge in a way which can in theory be pushed back upstream.